docs/filters.md

```diff
@@ -337,17 +337,19 @@ Test Case {#TestCase}
 Within the netcdf-c source tree, the directory __netcdf-c/nc_test4/filter_test__ contains
 a test case for testing dynamic filter writing and reading
 using bzip2. The test case is fragile and is only known to work for Linux
-and Cygwin. To avoid spurious failures, it must be explicitly
-invoked by entering the directory and issuing the command
-````
-make clean all makebzip2 check
-````
-Note that the Make target 'makebzip2' creates the bzip2 shared object.
-It is the target most likely to fail when executed on an untested or
+and Cygwin using automake. To avoid spurious failures, it must be explicitly
```

Comment:
What is fragile about the test case?

If it is fragile, maybe don't merge it until it is not fragile?